core/src/main/java/org/stellar/anchor/config/SecretConfig.java

```diff
@@ -16,4 +16,8 @@ public interface SecretConfig {
   String getDataSourceUsername();
 
   String getDataSourcePassword();
+
+  String getFireblocksApiKey();
```

Comment:
Can you create a separate bean for it?
E.g. `FireblocksSecretConfig`.
Ideally, `SecretConfig` should remain unchanged.